convert/src/vertex_to_bd/badgerdoc_format/plain_text_converter.py

```diff
@@ -10,8 +12,63 @@
 from ..models import BadgerdocToken, Offset, Page, PageSize
 
 
-def generate_chunks(text: str, size: int = 70) -> List[str]:
-    return [text[i : i + size] for i in range(0, len(text), size)]
+def generate_chunks(obj_to_split: List[str], size: int) -> List[List[str]]:
+    return [
+        obj_to_split[i : i + size] for i in range(0, len(obj_to_split), size)
+    ]
+
+
+class TextWrapper:
+    """
+    Split text to a separate lines. All whitespaces (including `\n`)
+    recognized as a ` ` and require one place.
+    """
+
+    # textwrap.TextWrapper is not suitable, because we need to convert
+    # it back to Vertex. Therefore, we need full control of process.
+
+    def __init__(self, line_length: int) -> None:
+        self.line_length = line_length
+
+    def pop_beginning_whitespaces(self, text: Deque[str]) -> List[str]:
+        wsp = []
+        while text and text[0] in string.whitespace:
+            wsp.append(text.popleft())
+        return wsp
+
+    def pop_next_word(self, text: Deque[str]) -> List[str]:
+        word = []
+        while text and text[0] not in string.whitespace:
+            word.append(text.popleft())
+        return word
+
+    def wrap(self, text: str) -> List[str]:
+        text = collections.deque(text)
+        lines: List[List[str]] = []
+        line: List[str] = self.pop_beginning_whitespaces(text)
+
+        while text:
+            if len(line) >= self.line_length:
+                lines.append(line)
+                line = []
+
+            word = self.pop_next_word(text)
+
+            if len(word) + len(line) <= self.line_length:
+                line.extend(word)
+            else:
```

Comment:
this way with first long word you will add first empty line

Reply:
just add the test
        tw = TextWrapper(line_length=10)
        text = "very_long_word word1 word2"

and fix the bug

Reply:
fixed